### PR TITLE
Init Azure.ResourceManager.SecurityCenter

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/autorest.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      $.defenderForServersAwsOffering.properties.vaAutoProvisioning.properties.configuration.properties.type['x-ms-enum']['name'] = 'VAAutoProvisioningType';
+      $.defenderForServersAwsOffering.properties.subPlan.properties.type['x-ms-enum']['name'] = 'AvailableSubPlanType';
+      $.defenderForServersGcpOffering.properties.vaAutoProvisioning.properties.configuration.properties.type['x-ms-enum']['name'] = 'VAAutoProvisioningType';
+      $.defenderForServersGcpOffering.properties.subPlan.properties.type['x-ms-enum']['name'] = 'AvailableSubPlanType';
+  - from: alerts.json
+    where: $.definitions
+    transform: >
+      $.ResourceIdentifier['x-ms-client-name'] = 'AlertResourceIdentifier';
+  # TODO: temporary remove these operations to mitigate the exception from BuildParameterMapping in Autorext.CSharp
+  - remove-operation: InformationProtectionPolicies_Get
```
Comment (Yao725) [inline sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/autorest.md:104]: Can we create an issue to track this problem?

**Review by Yao725 — APPROVED** [review]

File: sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/autorest.md (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/autorest.md:104]: https://github.com/Azure/azure-sdk-for-net/issues/30769

**Pull request merged**